index.yaml

```diff
@@ -37,6 +37,7 @@ distributions:
       default: kinetic/source-build.yaml
 doc_builds:
   independent-packages: doc-independent-build.yaml
+  independent-rosindex: doc-independent-rosindex.yaml
```

Comment:
We can probably drop `independent` from the name and just use rosindex which will produce the `doc_rosindex` job on the buildfarm.

Reply:
Ok, makes sense.